### PR TITLE
Update DB to add new fields required for planned features

### PR DESCRIPTION
**Pull request by mheon**
Also move some things around to try and make things more readable

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM, lets see if it passes tests.

**Comment by mheon** [comment]
Tests are green

File: libpod/sql_state.go [hunk 1]
```diff
@@ -271,10 +279,17 @@ func (s *SQLState) HasContainer(id string) (bool, error) {
 func (s *SQLState) AddContainer(ctr *Container) (err error) {
 	const (
 		addCtr = `INSERT INTO containers VALUES (
-                    ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?
+                    ?, ?, ?, ?, ?,
```
Comment (TomSweeneyRedHat) [inline libpod/sql_state.go]: I like this much better, easier to gauge changes and counts.

**Comment by TomSweeneyRedHat** [comment]
LGTM, but Travis tests still look to be cranking.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by mheon** [comment]
Image flake on RHCI, retrying

**Comment by mheon** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**